Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx

```diff
@@ -1,2829 +1,3872 @@
-    cooling->AddNode(pipeTube2, itube + 3, transformation);
-  }
-
-  // ------------------- Torus shape -------------------
-
-  // Sides Torus
-  TGeoVolume* pipeTorus1 =
-    gGeoManager->MakeTorus(Form("pipeTorus1_D2_H%d", half), mPipe, mRadius0, mRWater, mRWater + mDRPipe, 0., mAngle0);
-  pipeTorus1->SetLineColor(10);
-
-  for (Int_t itube = 0; itube < 3; itube++) {
-    rotation = new TGeoRotation("rotation", 180., 90., 0.);
-    transformation = new TGeoCombiTrans(mRadius0 + mXPosition0[itube], 0., -lMiddle / 2., rotation);
-    cooling->AddNode(pipeTorus1, itube, transformation);
-    rotation = new TGeoRotation("rotation", 180., -90., 0.);
-    transformation = new TGeoCombiTrans(mRadius0 + mXPosition0[itube], 0., lMiddle / 2., rotation);
-    cooling->AddNode(pipeTorus1, itube + 3, transformation);
-  }
-
-  // Central Torus
-  TGeoVolume* pipeTorus2 = gGeoManager->MakeTorus(Form("pipeTorus2_D2_H%d", half), mPipe, radius0mid, mRWater,
-                                                  mRWater + mDRPipe, -mAngle0, 2. * mAngle0);
-  pipeTorus2->SetLineColor(10);
-
-  for (Int_t itube = 0; itube < 3; itube++) {
-    rotation = new TGeoRotation("rotation", 0., 90., 0.);
-    transformation = new TGeoCombiTrans(mXPosition0[itube] + mRadius0 * (1 - (TMath::Cos(angle0rad))) +
-                                          mLpartial0 * TMath::Sin(angle0rad) - radius0mid * TMath::Cos(angle0rad),
-                                        0., 0., rotation);
-    cooling->AddNode(pipeTorus2, itube, transformation);
-  }
-
-  Double_t deltaz = mHeatExchangerThickness - mCarbonThickness * 2; // distance between pair of carbon plates
-
-  //  if (half == Top) {
-  //    rotation = new TGeoRotation ("rotation", 90., 90., 0.);
-  //    transformation = new TGeoCombiTrans(0., 0., mZPlan[disk] + deltaz/2. - mCarbonThickness - mRWater - mDRPipe,
-  //    rotation);
-  //    mHalfDisk->AddNode(cooling, 1, transformation);
-  //    transformation = new TGeoCombiTrans(0., 0., mZPlan[disk] - deltaz/2. + mCarbonThickness + mRWater + mDRPipe,
-  //    rotation);
-  //    mHalfDisk->AddNode(cooling, 2, transformation);
-  //  }
-  //  else if (half == Bottom) {
-  rotation = new TGeoRotation("rotation", -90., 90., 0.);
-  transformation =
-    new TGeoCombiTrans(0., 0., mZPlan[disk] + deltaz / 2. - mCarbonThickness - mRWater - mDRPipe, rotation);
-  mHalfDisk->AddNode(cooling, 3, transformation);
-  transformation =
-    new TGeoCombiTrans(0., 0., mZPlan[disk] - deltaz / 2. + mCarbonThickness + mRWater + mDRPipe, rotation);
-  mHalfDisk->AddNode(cooling, 4, transformation);
-  //  }
-
-  // **************************************** Carbon Plates ****************************************
-
-  auto* carbonPlate = new TGeoVolumeAssembly(Form("carbonPlate_D2_H%d", half));
-
-  auto* carbonBase2 = new TGeoBBox(Form("carbonBase2_D2_H%d", half), (mSupportXDimensions[disk][0]) / 2.,
-                                   (mSupportYDimensions[disk][0
```

Comment:
the encoding of the file was changed to `dos`, that's why it's completely different.

Reply:
What should I do? I'm working on files using a same emacs session. Is that an issue coming from my github interface instead?

Reply:
Should be fine now...

Reply:
Thanks